VolleyPaint/Assets/Scripts/Player/Shooting/GunController.cs

```diff
@@ -37,7 +37,7 @@ void Start()
 
     void Update()
     {
-        if (!IsOwner) return;
+        //if (!IsOwner) return;
```

Comment:
uncomment this or add a variable to toggle when this line is called

Reply:
Got it